homeassistant/components/roku/manifest.json

```diff
@@ -4,6 +4,14 @@
   "documentation": "https://www.home-assistant.io/integrations/roku",
   "requirements": ["roku==4.0.0"],
   "dependencies": [],
-  "after_dependencies": ["discovery"],
-  "codeowners": []
+  "ssdp": [
+    {
+      "st": "roku:ecp",
+      "manufacturer": "Roku",
+      "deviceType": "urn:roku-com:device:player:1-0"
+    }
+  ],
+  "codeowners": [],
```

Comment:
Can you add yourself as a code owner as you're making significant changes to this integration.

Reply:
@balloob #32695